lib/suspenders/generators/app_generator.rb

```diff
@@ -28,8 +28,11 @@ class AppGenerator < Rails::Generators::AppGenerator
     class_option :skip_test, type: :boolean, default: true,
       desc: "Skip Test Unit"
 
-    class_option :skip_system_test,
-      type: :boolean, default: true, desc: "Skip system test files"
+    class_option :css,
```

Comment:
This is the spirit of this pull-request, so we'll probably want a test for this.

This makes it so a new Rails app is generated via `--css=postcss`.